sign/sign.go

```diff
@@ -173,11 +174,23 @@ func (s *Signer) SignFile(path string) (*SignedObject, error) {
 		return nil, fmt.Errorf("sign file path: %s: %w", path, err)
 	}
 
-	object, err := s.impl.VerifyFileInternal(s, path)
-	if err != nil {
-		return nil, fmt.Errorf("verify file path: %s: %w", path, err)
+	// After signing, registry consistency may not be there right
+	// away. Retry the image verification if it fails
+	// ref: https://github.com/kubernetes-sigs/promo-tools/issues/536
+	i := 0
+	for {
```

Comment:
Let’s combine it into a classic `for i…`.

Reply:
Dropped this in favor of the exponential backoff call